example/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^0.1.3
+  cupertino_icons:
```

Comment:
I think the `cupertino_icons` can be completely removed from the example project since it's not used.